modules/core/db/update/mysql/19/190702-updateContentToReportTemplate.sql

```diff
@@ -0,0 +1 @@
+alter table REPORT_TEMPLATE change CONTENT CONTENT mediumblob^
```

Comment:
Will the template content save after altering column? Please check that

Reply:
Yes, I updated the database after modifying the scripts. Old data saved and column updated type.